beacon-chain/blockchain/process_block.go

```diff
@@ -514,12 +515,35 @@ func missingIndices(bs *filesystem.BlobStorage, root [32]byte, expected [][]byte
 	return missing, nil
 }
 
+func missingDataColumns(bs *filesystem.BlobStorage, root [32]byte, expected map[uint64]bool) (map[uint64]struct{}, error) {
+	if len(expected) == 0 {
+		return nil, nil
+	}
+	if len(expected) > int(params.BeaconConfig().NumberOfColumns) {
+		return nil, errMaxDataColumnsExceeded
+	}
+	indices, err := bs.Indices(root)
```

Comment:
Not sure it works as expected here: `indices` is of type `[fieldparams.MaxBlobsPerBlock]bool` and `fieldparams.MaxBlobsPerBlock == 6`.

Reply:
Ah, great catch will need to fix it